src/ConnectedKubernetes/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -54,12 +54,55 @@ function New-AzConnectedKubernetes {
         # The ID of the target subscription.
         ${SubscriptionId},
 
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.Category('Path')]
+        [System.Uri]
+        # The http URI of the proxy server for the kubernetes cluster to use
+        ${ProxyHttp},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.Category('Path')]
+        [System.Uri]
+        # The https URI of the proxy server for the kubernetes cluster to use
+        ${ProxyHttps},
```

Comment:
Why we need ProxyHttp and ProxyHttps at the same time?

Reply:
Thanks for the question.
The ProxyHttp, ProxyHttps are the alignment of az cli, see [doc1](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli%2Cazure-cloud#connect-using-an-outbound-proxy-server) [doc2](https://learn.microsoft.com/en-us/cli/azure/connectedk8s?view=azure-cli-latest#az-connectedk8s-connect-optional-parameters).
These settings will be propagated to helm charts later.

Reply:
Ok, I got your point. For PowerShell side, we need to keep singularity for parameter. How about using HttpsProxy, HttpProxy, NoProxy to keep align with https://learn.microsoft.com/en-us/powershell/azure/az-powershell-proxy?view=azps-9.4.0

Reply:
Got it! The parameters are renamed as your suggestion, thank you.